packages/api/cms-api/src/blocks/createSeoBlock.ts

```diff
@@ -51,7 +51,7 @@ interface CreateSeoBlockOptions<ImageBlock extends Block> {
     image?: ImageBlock;
 }
 
-// Block-factories need the their BlockInputInterface to be public
+// Block-factories need their BlockInputInterface to be public
```

Comment:
I'd rather remove the comment altogether. Obviously the the BlockInputInterface is not public 😁

Reply:
```suggestion
```